### ISSUE TITLE
Combine multiple pretrained models?

### ISSUE THREAD

**Issue by KDercksen** [KDercksen/hunter2_dbi#10]
Just another idea... We could use multiple pretrained networks instead of just Inception v3, and concatenate their bottleneck layers using `keras.layers.concatenate`. This way we could combine features from multiple models to (hopefully? maybe??) increase accuracy. Basic idea:

```python
from keras.layers import concatenate
inceptionv3 = inception_v3.InceptionV3(weights='imagenet', include_top=False)
resnet50 = resnet50.ResNet50(weights='imagenet', include_top=False)
base_output = concatenate([inceptionv3.output, resnet50.output])
# Add layers on top and train, maybe finetune top blocks etc
```
